src/Mvc/Mvc.ApiExplorer/test/DefaultApiDescriptionProviderTest.cs

```diff
@@ -1579,6 +1583,44 @@ public void GetApiDescription_ParameterDescription_FromQueryEmployee()
         Assert.Equal(typeof(string), id.Type);
     }
 
+    [Fact]
+    public void GetApiDescription_ParameterDescription_ParsableType()
```

Comment:
Do we have tests that cover nullable convertible types here? To validate the distinction between `UnderlyingOrModelType` and `ModelType`.

Reply:
Good catch, let me add them.